File: app/src/main/java/com/lms/mpasho_lms_news/models/Source.java
package com.lms.mpasho_lms_news.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Source {

    @SerializedName("name")
    @Expose
    private String name;

    public String getName() {
        return name;
    }


}

File: app/src/main/java/com/lms/mpasho_lms_news/models/Source.kt
package com.lms.mpasho_lms_news.models

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Source {

    @SerializedName("id")
    @Expose
    var id: String? = null

    @SerializedName("name")
    @Expose
    var name: String? = null
}